tests/test_holoclean_fusion.py

```diff
@@ -0,0 +1,14 @@
+import unittest
```

Comment:
This is here to test if passing in `entity_col` and `src_col` does the correct thing (at least in the DB). We don't actually check if it loads properly but it's a good temporary solution to integration test everything (to avoid programming errors like typos from creeping in).

Reply:
Ok, didn't notice that